### PR TITLE
Adding test_get_events

### LINKED ISSUE #24: Test Get Events API
**Is your feature request related to a problem? Please describe.**
We need to test the Get Events API refactored in #20 

**Describe the solution you'd like**
Relevant tests must be added to `tests/events/tests_get_events_api.py`.

**Additional context**
Few suggestions for how the tests could be are as follows:
* Invalid Input (e.g. `twentynineteen`)
* Fetch empty event list
* Fetch list with multiple events
* etc.

### PR DESCRIPTION
**Pull request by akhil-maker**
### Description
Adding test cases for get_events_api : Various tests are added for this 
                                             - test for many events
                                             - test for one event
                                             - test for no events (empty list)
                                             - test with invalid input 

**Type of Change:** (Delete irrelevant options)
- Testing

**Code/Quality Assurance Only** (Delete irrelevant options)

- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
Tested locally with all test cases

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [ ] Update Swagger documentation and the exported file at /docs folder
- [ ] Update requirements.txt

**Code/Quality Assurance**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Review by anshsrtv — APPROVED** [review]

**Comment by anshsrtv** [comment]
Pull from upstream and restart the build in Travis CI.

File: tests/events/test_get_events.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            details="test_details",
+            details_html="test.html",
+            cover_pic="test_cover.jpeg",
+            icon="test_icon.png",
+            email="test@email.com",
+            flag=True,
+            year=2020,
+        )
+
+    def test_pass_get_mentors_many(self):
```
Comment (priyanshisharma) [inline tests/events/test_get_events.py:64]:
```suggestion
    def test_pass_get_events_many(self):
```

**Review by priyanshisharma — COMMENTED** [review]
Looks good to me 🔥 
Just a little suggestion below, and pull from upstream, since it is a PR with tests, I want the travis build to pass (we've merged the changes in the main repo that's why you need to pull them). 😄

File: tests/events/test_get_events.py (hunk 1 above)
Comment (akhil-maker) [reply tests/events/test_get_events.py:64]: ohh!! yeah sure..

**Review by priyanshisharma — APPROVED** [review]
Works well 🚀 
PS - Including a missing environment variable in this PR 😄 🙌

**Comment by akhil-maker** [comment]
> Works well 🚀
> PS - Including a missing environment variable in this PR 😄 🙌

Thank u @priyanshisharma ma'am

**Pull request merged**